src/ruby/symengine.gemspec

```diff
@@ -0,0 +1,16 @@
+Gem::Specification.new do |gem|
+	gem.name = 'symengine'
+	gem.version = '0.0.0.pre'
+	gem.summary = "Ruby interface for SymEngine, a fast C++ library for symbolic manipulation"
+	gem.description = "This gem provides a Ruby interface for SymEngine, a fast C++ library for symbolic manipulation, so that the you can use the features of SymEngine in ruby."
+	gem.authors = ['Abinash Meher']
+	gem.email = ["abinashdakshana999@gmail.com"]
+	gem.homepage = 'https://github.com/sympy/symengine'
+	gem.files = Dir["lib/**/*", "bin/*", "../../LICENSE", "*.md"]
```

Comment:
I've got an error while installing the gem:
```
ERROR:  While executing gem ... (Gem::Package::PathError)
    installing into parent path /var/lib/gems/2.1.0/LICENSE of /var/lib/gems/2.1.0/gems/symengine-0.0.0.pre is not allowed
```
I think LICENSE file should be copied into ruby wrapper folder and
```
gem.files = Dir["lib/**/*", "bin/*", "../../LICENSE", "*.md"]
```
will became
```
gem.files = Dir["lib/**/*", "bin/*", "LICENSE", "*.md"]
```
Apart of this installation is succesful on ubuntu 14.04.